lib/rules/no-extra-parens.js

```diff
@@ -428,6 +429,38 @@ module.exports = {
             }
         }
 
+        /**
+         * Check the parentheses around the super class of the given class definition.
+         * @param {ASTNode} node The node of class declarations to check.
+         * @returns {void}
+         */
+        function dryClass(node) {
```

Comment:
Maybe these functions could be called `"checkClass"` and `"checkSpreadOperator"` instead? I'm not sure the `dry` prefix is clear. Sorry if I misunderstood its meaning.